doc/sphinx-guides/source/installation/config.rst

```diff
@@ -975,9 +976,13 @@ Note that in either case, when using the ``sequentialNumber`` option, datasets a
 :FilePIDsEnabled
 ++++++++++++++++
 
-Enable/disable the publishing of file based PIDs for the whole installation. This is enabled by default
+Toggles publishing of file-based PIDs for the entire installation. By default this setting is absent and Dataverse assumes it to be true.
 
-``curl -X PUT -d 'true' http://localhost:8080/api/admin/settings/:FilePIDsEnabled``
+If you don't want to register file-based PIDs for your installation, set:
+
+``curl -X PUT -d 'false' http://localhost:8080/api/admin/settings/:FilePIDsEnabled``
+
+Note: File-level PID registration was added in 4.9 and is required until version 4.9.3.
```

Comment:
This all makes sense to me. Much more clear and a better example. We **could** even add a note about file PIDs under "Decisions to Make" in a future version of http://guides.dataverse.org/en/4.9.4/installation/prep.html#decisions-to-make . It sounds like you don't like the default, out of the box setting to have file PIDs on.

Reply:
It's not that I disliked it, but rather the documentation describes the setting as optional. I found that, in its absence Dataverse assumes the setting to be true, so it's only-optional-if-you-disable-it. I was trying to make that more clear in the config page (Odum got 7 file PIDs by accident; I hadn't expected them as I hadn't purposefully set it to true)

I can update the verbage about registering DOIs, which as you point out lags behind recent changes in the DOI landscape.

Reply:
The default is a change in behavior, which I can see as being potentially confusing.